numpy/core/src/umath/ufunc_object.c

```diff
@@ -3842,10 +3842,13 @@ PyUFunc_GenericReduction(PyUFuncObject *ufunc, PyObject *args,
     PyArray_Descr *otype = NULL;
     PyArrayObject *out = NULL;
     int keepdims = 0;
-    static char *kwlist1[] = {"array", "axis", "dtype",
-                                "out", "keepdims", NULL};
-    static char *kwlist2[] = {"array", "indices", "axis",
-                                "dtype", "out", NULL};
+    static char *reduce_kwlist[] = {
+        "array",            "axis", "dtype", "out", "keepdims", NULL};
```

Comment:
That's a set of spaces to align columns, to make visually diffing the arg lists easier.

The previous format was just indented with a bunch of soft tabs until it roughly aligned. I definitely think that having all the list items on one line is an improvement, but could be persuaded to remove that alignment